### ISSUE TITLE
//Undo not working properly/at all

### ISSUE THREAD

**Issue by Rezois** [IntellectualSites/FastAsyncWorldEdit#786]
**[REQUIRED] FastAsyncWorldEdit Configuration Files** 

**/fawe debugpaste**: https://athion.net/ISPaster/paste/view/9f5293270cb04446a372a57bb1d70065

**Required Information**
- FAWE Version Number (`/version FastAsyncWorldEdit`):     1.16-474;e08fda0
- Spigot/Paper Version Number (`/version`):     git-Paper-330 (MC: 1.16.4)
- Minecraft Version:     1.16.4

**Describe the bug**
Currently after any fawe action, when I type //undo it says "Undid 1 available edits" without doing anything. Doing this again provides the same message in chat with this error https://hastebin.com/fidadanoyo.yaml

Sometimes it also will work after a second //undo or //redo, but only after using the command twice, I shouldn't have to use the command twice for it to work so something is still wrong

And also sometimes only part of the action will undo/redo

EDIT: Seems to be almost 100% caused by inventory mode 1, no clue how

**To Reproduce**
1. Make a grief prevention claim and have free build inventory mode enabled
2. Do any action with fawe. I usually do //set or //replace
3. Do //undo

**Plugins being used on the server**
Plugins (38): AAC5, AACAdditionPro, ArmoredElytra, AutoCrafters, BanFromClaim, BestTools, BetterConcrete, BlepRevival, ChestsPlusPlus, Chunky, ClickDye, CMI, CMIPaperLib, ContainerPassthrough, CoreProtect, CraftBook, CropReplace, FastAsyncWorldEdit (WorldEdit), FastCraft, FasterVillagers, FastLeafDecay, GeoFurnace, GPTeleport, GriefPrevention, IronDoors*, LeashTeleport, LuckPerms, MyPet, PortableHorses, ProtocolLib, SheepColorSwapper, SimpleNoCropTrample*, StreetLamps, SurvivalInvisiframes, TradePlus, TridentTweaks, VeinMiner, WorldGuard
(This was happening before AAC5 and AACAP)

**Checklist**:

- [x] I included all information required in the sections above
- [x] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit/issues?q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of [FastAsyncWorldEdit for 1.16.4](https://ci.athion.net/job/FastAsyncWorldEdit-1.16/)
- [x] I made sure the bug/error is not caused by any other plugin

**Comment by dordsor21** [comment]
Fixed
